File: coresuite-dotnet-core-console-vb/merger/FormFlattening.vb
Imports ceTe.DynamicPDF.Forms
Imports ceTe.DynamicPDF.Merger

Public Class FormFlattening

    Public Shared Sub Run(outputPdfPath As String)

        Dim document As MergeDocument = New MergeDocument(Util.GetResourcePath("PDFs/fw9AcroForm_18_filled.pdf"))

        document.Form.Output = FormOutput.Flatten

        document.Draw(outputPdfPath)
    End Sub
End Class

File: coresuite-dotnet-framework-console-cs/merger/FormFlattening.cs
using ceTe.DynamicPDF.Forms;
using ceTe.DynamicPDF.Merger;

namespace coresuite_dotnet_framework_console_cs.Merger
{
    public class FormFlattening
    {
        public static void Run(string outputPdfPath)
        {
            MergeDocument document = new MergeDocument(Util.GetResourcePath("PDFs/fw9AcroForm_18_filled.pdf"));

            document.Form.Output = FormOutput.Flatten;

            document.Draw(outputPdfPath);
        }
    }
}